skyplane/cli/cli.py

```diff
@@ -413,6 +443,45 @@ def init(
 
     cloud_config.to_config_file(config_path)
     typer.secho(f"\nConfig file saved to {config_path}", fg="green")
+
+    # Set metrics collection by default
+    usage_stats.show_usage_stats_prompt()
+    return 0
+
+
+@app.command()
+def metrics(
```

Comment:
It's minor but can we potentially fold this into the `skyplane config` CLI as a regular flag like `multipart_enabled`?

Reply:
moved!